save-backend/src/main/kotlin/com/saveourtool/save/backend/configs/ApplicationConfiguration.kt

```diff
@@ -13,4 +16,11 @@ import org.springframework.web.reactive.config.EnableWebFlux
 @EntityScan("com.saveourtool.save.entities")
 @ImportAutoConfiguration(HibernateMetricsAutoConfiguration::class)
 @Suppress("MISSING_KDOC_TOP_LEVEL")
-class ApplicationConfiguration
+class ApplicationConfiguration {
+    /**
+     * @param configProperties
+     * @return [LokiLogService] if [ConfigProperties.lokiServiceUrl] is provided, otherwise [LogService.stub]
+     */
+    @Bean
+    fun logService(configProperties: ConfigProperties): LogService = configProperties.lokiServiceUrl?.let { LokiLogService(it) } ?: LogService.stub
```

Comment:
Maybe `@ConditionalOnProperty`? Though considering we can't get property name from the variable name, it would make little sense

Reply:
yeap, sandbox registers it using `@ConditionalOnProperty`